x/ecocredit/module/module.go

```diff
@@ -59,16 +65,19 @@ func (a AppModule) ExportGenesis(sdk.Context, codec.JSONMarshaler) json.RawMessa
 
 func (a AppModule) RegisterInvariants(sdk.InvariantRegistry) {}
 
-func (a AppModule) Route() sdk.Route { return sdk.Route{} }
+func (a AppModule) Route() sdk.Route {
+	return sdk.Route{}
+}
 
-func (a AppModule) QuerierRoute() string { return "" }
+func (a AppModule) QuerierRoute() string { return a.Name() }
```

Comment:
Unneeded. This is legacy stuff

Reply:
moved to a deprecated section of the file.